.circleci/config.yml

```diff
@@ -1,11 +1,11 @@
 ---
 version: 2.1
 orbs:
-  lsk-python: lsk/python@1.2
+  lsk-python: lsk/python@1.4
```

Comment:
Had to bump to fix this: https://app.circleci.com/pipelines/github/lightspeed-hospitality/pre-commit-hooks/268/workflows/8a807644-30e0-44ea-85c0-411f058e4e6b/jobs/229